src/pat/structure/js/views/app.js

```diff
@@ -626,9 +623,13 @@ export default BaseView.extend({
                 .after(this.uploadView.render().el);
         }
 
-        await this.tableView.render();
-        this.$el.append(this.tableView.el);
-        this.$el.append(this.pagingView.render().el);
+        const tableView = new TableView({ app: this });
+        await tableView.render()
+        this.$el.append(tableView.el);
+
+        const pagingView = new PagingView({ app: this });
+        pagingView.render()
```

Comment:
the pagingview is probably not async, therefore no await, right?

Reply:
there's no async in PagingView